client/docs/transformations.md

```diff
@@ -0,0 +1,64 @@
+# Transformations
+
+## Local Provider
+
+### Dataframe Transformation
+::: featureform.register.LocalProvider.df_transformation
+    handler: python
+    options:
+        show_root_heading: false
+        show_source: false
+        show_root_toc_entry: false
+
+### SQL Transformation
+::: featureform.register.LocalProvider.sql_transformation
+    handler: python
+    options:
+        show_root_heading: false
+        show_source: false
+        show_root_toc_entry: false
+
+## SQL Providers
+
+::: featureform.register.OfflineSQLProvider.sql_transformation
+    handler: python
+    options:
+        show_root_heading: false
+        show_source: false
+        show_root_toc_entry: false
+
+## Spark
+
+### SQL Transformation
+::: featureform.register.OfflineSparkProvider.sql_transformation
+    handler: python
+    options:
+        show_root_heading: false
+        show_source: false
+        show_root_toc_entry: false
+
+### Dataframe Transformation
+::: featureform.register.OfflineSparkProvider.df_transformation
+    handler: python
+    options:
+        show_root_heading: false
+        show_source: false
+        show_root_toc_entry: false
+
+## Kubernetes Pandas Runner
+
+### SQL Transformation
+::: featureform.register.OfflineK8sProvider.sql_transformation
+    handler: python
+    options:
+        show_root_heading: false
+        show_source: false
+        show_root_toc_entry: false
+
+### Dataframe Transformation
+::: featureform.register.OfflineK8sProvider.df_transformation
```

Comment:
It might be confusing to go from `k8s.sql_transformation(...)` to `k8s_azure.df_transformation(...)`. I would recommend we use `k8s` for both.